.gitignore

```diff
@@ -10,3 +10,6 @@ tmp/
 
 # This file gets generated as part of `make run-local`
 operator.log
+
+# GoLand project files
+.idea
```

Comment:
Let's keep IDE/editor specific settings out of this file. Better add this to your personal config, either [globally](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) or [per repo](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file).

Reply:
Fixed it. Thank you.